src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -83,44 +64,33 @@ final case class PurposeApiServiceImpl(
     contexts: Seq[(String, String)],
     toEntityMarshallerProblem: ToEntityMarshaller[Problem],
     toEntityMarshallerFile: ToEntityMarshaller[File]
-  ): Route = {
+  ): Route = { // TODO This endpoint does not contain authz on role, is it ok?
```

Comment:
does any role is allowed to access this endpoint?

Reply:
At least the admin role should probably be added.

Reply:
@beetlecrunch this is the retrieve of the risk analysis document, are admins the only ones allowed?

Reply:
Yes, security and api roles do not access to purpose section at all.